core/src/main/res/values/strings.xml

```diff
@@ -30,6 +30,7 @@
     <string name="episode_cache_full_message">The episode cache limit has been reached. You can increase the cache size in the Settings.</string>
     <string name="playback_statistics_label">Playback</string>
     <string name="download_statistics_label">Downloads</string>
+    <string name="notification_pref_fragment">Notification Manager</string>
```

Comment:
I would rather call this simply "Notifications" (not manager).

Reply:
changed the name to Notifications